### PR TITLE
Added crossplane-azure-upbound Helm chart - New PR

### PR DESCRIPTION
**Pull request by paolosalvatori**
I added a new Helm chat in the crossplane-azure-upbound folder to install the Crossplane upbound packages for Azure. The ProviderConfig CRD supports both the service principal and managed identity credentials.

### CODE REVIEW

File: charts/crossplane-azure-upbound/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 2.0.2
```
Comment (csantanapr) [inline charts/crossplane-azure-upbound/Chart.yaml]: any reason you didn't make version and appVersion 1.0.0?

Comment (csantanapr) [reply charts/crossplane-azure-upbound/Chart.yaml]: This will avoid people wondering where is the previous version 1.x, if the first release is 2.x

Comment (paolosalvatori) [reply charts/crossplane-azure-upbound/Chart.yaml]: Done 😸 I had the same thought by the way. I changed the version and appVersion to 1.0.0

**Review by csantanapr — APPROVED** [review]
Thank you for the PR!

**Pull request merged**